OneCommunityGlobal/HighestGoodNetworkApp
# Jerry - Unit test NewRolePopUp.jsx

# Description
Create unit tests for `src/components/PermissionsManagement/NewRolePopUp.jsx`

## Related PRS (if any):
N/A

## Main changes explained:
- Added a new file: `src/components/PermissionsManagement/__tests__/NewRolePopUp.test.js`

## How to test:
1. On the `development` branch of HGNRest (backend), run the server by running the following command: `npm i && npm run build && npm start`
2. check into current branch on HighestGoodNetworkApp (frontend).
3. do `npm install`
4. Run `npm test NewRolePopUp.test.js` to run the unit tests


## Screenshots or videos of changes:

<img width="582" alt="Screenshot 2024-03-17 at 6 21 01 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/55968519/134ab673-a48b-407c-84a9-3da414790b45">
